src/Illuminate/View/Compilers/Concerns/CompilesFragments.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Illuminate\View\Compilers\Concerns;
+
+trait CompilesFragments
+{
+    /**
+     * Compile the fragment statements into valid PHP.
+     *
+     * @param  string  $expression
+     * @return string
+     */
+    protected function compileFragment($expression)
+    {
+        $this->lastFragment = trim($expression, "()'\" ");
```

Comment:
@brunoalod assigning to non-existing properties is deprecated in PHP 8.2

Reply:
Missed that one. Would you like me to submit a PR?

Reply:
Just saw you sent a PR fixing this. Thanks 👍